apps/linode-marketplace-openlitespeed-django/roles/post/tasks/main.yml

```diff
@@ -0,0 +1,13 @@
+-
+# motd and creds gen
+- name: copy MOTD template to /etc/motd
+  template:
+    src: templates/motd.j2
+    dest: /etc/motd
+
+- name: writing openlitespeed creds into file
+  copy:
+    dest: /root/.credentials
```

Comment:
creds are now located in /home/$username/.credentials

Reply:
Check.